src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
Just checked this, you're right. I think that's fine though, as I would not deploy the `install.php` in such a case anyway, which means the route will be available and work just fine.